amplitude/Amplitude-TypeScript
# feat: persist last event id - to be used on next initialization

### Summary

* Persist last event it used in storage

### Testing

* Automated tests are passing
* Manually tested by initializing SDK on a website
  * decode cookies and verify event id is persisted
  * re-initialize SDK on a different page load and verify that event id is incremental across page loads

### Checklist

* [x] Does your PR title have the correct [title format](https://github.com/amplitude/Amplitude-TypeScript/blob/main/CONTRIBUTING.md#pr-commit-title-conventions)?
* Does your PR have a breaking change?:  No
